README.md

```diff
@@ -7,23 +7,24 @@ Zowe Explorer is a separately delivered component of Zowe belonging to the [Zowe
 
 The Zowe Explorer extension modernizes the way developers and system administrators interact with z/OS mainframes. Working with data sets and USS files from VSC can be more convenient than using 3270 emulators, and complements your Zowe CLI experience. The extension provides the following benefits:
 
-* Enables developers to create, modify, rename, copy and upload data sets directly to a z/OS mainframe.
-* Enables developers to create, modify, rename and upload USS files directly to a z/OS mainframe.
+* Enables you to create, modify, rename, copy and upload data sets directly to a z/OS mainframe.
+* Enables you to create, modify, rename and upload USS files directly to a z/OS mainframe.
 * Provides a more streamlined way to access data sets, uss files and jobs.
-* Lets you create and use Zowe CLI `zosmf` compatible profiles.
+* Lets you create, edit and delete Zowe CLI `zosmf` compatible profiles.
+* Lets you use Secure Credential Store plug-in to ensure your credentials security.
```

Comment:
Lets you use the Secure Credential Store plug-in to store your credentials securely in the settings.